ReleaseTooling/Sources/ZipBuilder/main.swift

```diff
@@ -232,6 +238,11 @@ struct ZipBuilderTool: ParsableCommand {
                                macos: minimumMACOSVersion,
                                tvos: minimumTVOSVersion)
 
+    // Update iOS target platforms if `--skip-catalyst` was specified.
+    if skipCatalyst {
+      SkipCatalyst.set()
+    }
+
```

Comment:
Hmm can this be passed in some other way other then having a global flag? This API feels awkward

Reply:
Maybe? But I couldn't come up with one. There is a similar API for minimum versions. Any suggestions?